crates/wasmi/src/store.rs

```diff
@@ -67,6 +68,22 @@ impl StoreIdx {
 /// A stored entity.
 pub type Stored<Idx> = GuardedEntity<StoreIdx, Idx>;
 
+pub struct ResourceLimiterRef<'a>(pub(crate) Option<&'a mut (dyn crate::ResourceLimiter)>);
```

Comment:
This is just personal preference but I would prefer to have no `pub(crate)` for the field in the `ResourceLimiterRef` type and instead a convenience getter:
```rust
impl<'a> ResourceLimiterRef<'a> {
    pub fn as_resource_limiter(&mut self) -> &mut Option<&'a mut dyn crate::ResourceLimiter> {
        &mut self.0
    }
}
```
Then all instances of `&mut limiter.0` can be replaced with `limiter.as_resource_limiter()`.

Reply:
Sure.